questionary/prompts/common.py

```diff
@@ -116,6 +116,8 @@ def build(c: Union[str, "Choice", Dict[str, Any]]) -> "Choice":
             return c
         elif isinstance(c, str):
             return Choice(c, c)
+        elif isinstance(c, (int, bool, float)):  # Ajout pour les types primitifs
```

Comment:
looks good, do you mind changing the functions comment as well, stating that these types are allowed as well?

please remove frensh comments